src/screens/PostScreen.tsx

```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
 export default function PostScreen() {
   return (
     <View style={styles.container}>
-      <ProfilePlaceholder />
-      <Text>rbeggs</Text>
-      <Text>September 19</Text>
-      <Text>
+      {/* Profile Header */}
+      <View style={styles.profileHeader}>
+        <ProfilePlaceholder style={styles.profileIcon} />
+        <View>
+          <Text style={styles.profileName}>rbeggs</Text>
+          <Text style={styles.postDate}>September 19</Text>
+        </View>
+      </View>
+
+      {/* Post Content */}
+      <Text style={styles.postText}>
         In response to the growing homelessness crisis in San Francisco, a local
         nonprofit organization, Code Tenderloin, has launched a comprehensive
         initiative aimed at providing long-term solutions for individuals
         experiencing homelessness. The organization, founded in 2015, is
         dedicated to addressing both immediate needs and underlying causes of
         homelessness through a combination of shelter services, job training
-        programs, and mental health support. read more online:
-        https://www.codetenderloin.org/
-      </Text>
-      <Text>
-        Image URL:
-        'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg'
+        programs, and mental health support.
       </Text>
-      <HeartIcon />
-      <Text>256 Likes</Text>
-      <ShareIcon />
-      <ProfilePlaceholder />
-      <Text>philip_ye</Text>
-      <Text>September 20</Text>
-      <Text>
-        This organization is doing amazing work tackling the complex root causes
-        of the issue.
+
+      {/* External Link */}
+
+      <Text style={styles.linkText}>
+        read more online: https://www.codetenderloin.org/
       </Text>
-      <ProfilePlaceholder />
-      <Text>vppraggie</Text>
-      <Text>September 21</Text>
-      <Text>Thanks for sharing!</Text>
+
+      {/* Post Image */}
+      <Image
+        style={styles.postImage}
+        source={{
+          uri: 'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg',
+        }}
+      />
+
+      {/* Like and Share Section */}
+      <View style={styles.interactionRow}>
+        <HeartIcon style={styles.icon} />
+        <Text style={styles.likesText}>256 Likes</Text>
+        <ShareIcon style={styles.icon} />
+      </View>
+
+      {/* Comments */}
+      <View style={styles.commentSection}>
+        <View style={styles.comment}>
+          <ProfilePlaceholder style={styles.commentIcon} />
+          <View>
+            <Text style={styles.commentName}>philip_ye</Text>
+            <Text style={styles.commentText}>
+              This organization is doing amazing work tackling the complex root
+              causes of the issue.
```

Comment:
How could we integrate the comment text here? Try placing the username and date text in the same div, that way they appear inline, above the comment text.